tests/unit/scripts/test_lint.py

```diff
@@ -0,0 +1,314 @@
+        abort if timeout handler fails = True
+        abort if stalled handler fails = True
+        abort if inactivity handler fails = False
+        aborted handler = woo
+        stalled handler = bar
+        timeout handler = bas
+        shutdown handler = qux
+        startup handler = now
+        inactivity handler = bored
+        mail to = eleanor.rigby@beatles.lv
+        mail from = fr.mckenzie@beatles.lv
+        mail footer = "Collecting The Rice"
+        mail smtp = 123.456.789.10
+        timeout = 30
+        inactivity = 30
+        abort on inactivity = 30
+    [[parameters]]
+    [[parameter templates]]
+    [[mail]]
+        task event mail interval    = PT4M # deliberately added lots of spaces.
+
+[scheduling]
+    max active cycle points = 5
+    hold after point = 20220101T0000Z
+    [[dependencies]]
+        [[[R1]]]
+            graph = foo
+
+
+
+[runtime]
+    [[MYFAM]]
+        extra log files = True
+        {% from 'cylc.flow' import LOG %}
+        script = {{HELLOWORLD}}
+        [[[suite state polling]]]
+            template = and
+        [[[remote]]]
+            host = parasite
+            suite definition directory = '/home/bar'
+        [[[job]]]
+            batch system = slurm
+            shell = fish
+        [[[events]]]
+            mail retry delays = PT30S
+            warning handler = frr.sh
+            submission timeout handler = faiuhdf
+            submission retry handler = vhbayhrbfgau
+            submission failed handler = giaSEHFUIHJ
+            execution timeout handler = sdfghjkl
+            expired handler = dafuhj
+            late handler = dafuhj
+            submitted handler = dafuhj
+            started handler = dafuhj
+            succeeded handler = dafuhj
+            custom handler = efrgh
+            critical handler = fgjdsfs
+            retry handler = dfaiuhfrgpa
+            sumbission handler = fas9hrfgaiuph
+
+"""
+
+
+LINT_TEST_FILE = """
+\t[scheduler]
+
+ [scheduler]
+
+[[dependencies]]
+
+[runtime]
+          [[foo]]
+        inherit = hello
+     [[[job]]]
+something\t
+"""
+
+LINT_TEST_FILE += (
+    '\nscript = the quick brown fox jumps over the lazy dog '
+    'until it becomes clear that this line is far longer the 79 characters.')
+
+
+@pytest.fixture()
+def create_testable_file(monkeypatch, capsys):
+    def _inner(test_file, checks):
+        monkeypatch.setattr(Path, 'read_text', lambda _: test_file)
+        check_cylc_file(Path('x'), parse_checks(checks))
+        return capsys.readouterr()
+    return _inner
+
+
+@pytest.mark.parametrize(
+    'number', range(1, len(UPG_CHECKS) + 1)
+)
+def test_check_cylc_file_7to8(create_testable_file, number, capsys):
+    # try:
+    result = create_testable_file(TEST_FILE, '728').out
+    assert f'[U{number:03d}]' in result
+    # except AssertionError:
+    #     raise AssertionError(
+    #         f'missing error number U{number:03d}'
+    #         f'{[*CHECKS["U"].keys()][number]}'
+    #     )
```

Comment:
What is to happen to these commented lines?

Reply:
Sometimes it's easier to debug this test without them - I forgot to put them back.